project/Dependencies.scala

```diff
@@ -6,7 +6,7 @@ import scala.collection.Seq
 import sbt._
 
 object Dependencies {
-  private val http4sVersion = "0.18.13"
+  private val http4sVersion = "0.19.0-M2"
```

Comment:
I think we can upgrade to `0.19.0-M4`

Reply:
Unfortunately it's no as straightforward as bumping to `0.19.0-M4`. The latest milestones break even more of the client API. Working on it.